api/swagger.yml

```diff
@@ -1770,6 +1770,9 @@ components:
               merged_commit_id:
                 type: string
                 description: the commit id of merged PRs
+              close_date:
```

Comment:
Please add description, so it will be clear that it's relevant also for merged PRs, not only closed ones.

Reply:
Also, if we have `merged_commit_id` - 
Shouldn't this be `closed_date`?